src/web/lib/components/ThemeColorsEditor/index.scss

```diff
@@ -46,7 +70,7 @@
     dd {
       display: none;
       position: absolute;
-      left: 55%;
+      left: 47.5%;
```

Comment:
The CSS could probably use a little help to align these buttons with the color editor: I tried tweaking the position of the theme color editor, since it seemed a little squashed on my screen. But, that doesn't seem to have helped. I might give it another poke tomorrow, but meh